spec/mailers/previews/subscribe_preview.rb

```diff
@@ -0,0 +1,4 @@
+# Preview all emails at http://localhost:3000/rails/mailers/subscribe
+class SubscribePreview < ActionMailer::Preview
```

Comment:
is this file useful?

Reply:
auto generated andwe don't need it so far , so i will remove it